<<<<<<< HEAD:venv/Lib/site-packages/numpy/typing/tests/data/fail/datasource.pyi
from pathlib import Path
import numpy as np

path: Path
d1: np.lib.npyio.DataSource

d1.abspath(path)  # E: incompatible type
d1.abspath(b"...")  # E: incompatible type

d1.exists(path)  # E: incompatible type
d1.exists(b"...")  # E: incompatible type

d1.open(path, "r")  # E: incompatible type
d1.open(b"...", encoding="utf8")  # E: incompatible type
d1.open(None, newline="/n")  # E: incompatible type
=======
from pathlib import Path
import numpy as np

path: Path
d1: np.lib.npyio.DataSource

d1.abspath(path)  # E: incompatible type
d1.abspath(b"...")  # E: incompatible type

d1.exists(path)  # E: incompatible type
d1.exists(b"...")  # E: incompatible type

d1.open(path, "r")  # E: incompatible type
d1.open(b"...", encoding="utf8")  # E: incompatible type
d1.open(None, newline="/n")  # E: incompatible type
>>>>>>> 88e2e56b8b4d2bdf087a4d11249f6993dbacf836:env/Lib/site-packages/numpy/typing/tests/data/fail/datasource.pyi
