<<<<<<< HEAD:venv/Lib/site-packages/numpy/typing/tests/data/fail/dtype.pyi
import numpy as np


class Test1:
    not_dtype = np.dtype(float)


class Test2:
    dtype = float


np.dtype(Test1())  # E: No overload variant of "dtype" matches
np.dtype(Test2())  # E: incompatible type

np.dtype(  # E: No overload variant of "dtype" matches
    {
        "field1": (float, 1),
        "field2": (int, 3),
    }
)
=======
import numpy as np


class Test1:
    not_dtype = np.dtype(float)


class Test2:
    dtype = float


np.dtype(Test1())  # E: No overload variant of "dtype" matches
np.dtype(Test2())  # E: incompatible type

np.dtype(  # E: No overload variant of "dtype" matches
    {
        "field1": (float, 1),
        "field2": (int, 3),
    }
)
>>>>>>> 88e2e56b8b4d2bdf087a4d11249f6993dbacf836:env/Lib/site-packages/numpy/typing/tests/data/fail/dtype.pyi
