<<<<<<< HEAD:venv/Lib/site-packages/numpy/typing/tests/data/fail/lib_utils.pyi
import numpy.lib.array_utils as array_utils

array_utils.byte_bounds(1)  # E: incompatible type
=======
import numpy.lib.array_utils as array_utils

array_utils.byte_bounds(1)  # E: incompatible type
>>>>>>> 88e2e56b8b4d2bdf087a4d11249f6993dbacf836:env/Lib/site-packages/numpy/typing/tests/data/fail/lib_utils.pyi
