<<<<<<< HEAD:venv/Lib/site-packages/numpy/typing/tests/data/fail/nditer.pyi
import numpy as np

class Test(np.nditer): ...  # E: Cannot inherit from final class

np.nditer([0, 1], flags=["test"])  # E: incompatible type
np.nditer([0, 1], op_flags=[["test"]])  # E: incompatible type
np.nditer([0, 1], itershape=(1.0,))  # E: incompatible type
np.nditer([0, 1], buffersize=1.0)  # E: incompatible type
=======
import numpy as np

class Test(np.nditer): ...  # E: Cannot inherit from final class

np.nditer([0, 1], flags=["test"])  # E: incompatible type
np.nditer([0, 1], op_flags=[["test"]])  # E: incompatible type
np.nditer([0, 1], itershape=(1.0,))  # E: incompatible type
np.nditer([0, 1], buffersize=1.0)  # E: incompatible type
>>>>>>> 88e2e56b8b4d2bdf087a4d11249f6993dbacf836:env/Lib/site-packages/numpy/typing/tests/data/fail/nditer.pyi
