<<<<<<< HEAD:venv/Lib/site-packages/numpy/typing/tests/data/fail/numerictypes.pyi
import numpy as np

np.isdtype(1, np.int64)  # E: incompatible type

np.issubdtype(1, np.int64)  # E: incompatible type
=======
import numpy as np

np.isdtype(1, np.int64)  # E: incompatible type

np.issubdtype(1, np.int64)  # E: incompatible type
>>>>>>> 88e2e56b8b4d2bdf087a4d11249f6993dbacf836:env/Lib/site-packages/numpy/typing/tests/data/fail/numerictypes.pyi
